ahmed-yara-sara/src/components/Navbar.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import "react-bootstrap";
+import { Navbar, Nav } from "react-bootstrap";
+import Search from "./Search";
+import Dropdown from "./Dropdown"
+import "bootstrap/dist/css/bootstrap.min.css";
+
+export default function Navbar1(props) {
+  return (
+    <>
+      <Navbar bg="dark" variant="dark">
+        <Navbar.Brand href="#home">Movies For You</Navbar.Brand>
+        <Nav className="mr-auto">
+        </Nav>
+        <Dropdown></Dropdown>
```

Comment:
```suggestion
        <Dropdown />
```

change all components to look like this